src/pages/taejoon/List/List.scss

```
@@ -0,0 +1,89 @@
+
+
+.container-List {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    
+}
+.header {
+    background-color: 	#f4f3ec;
+    display: flex;
+    justify-content: space-between;
+    width: 90%;
+    margin-left: 5%;
+    
+}
+.main_logo {
+    font-family: 'Dancing Script', cursive;
+    font-size: 1.5rem;
+    text-decoration: none;
+    color: black;
+    padding: 1% 2% 1% 2%;
+
+}
+
+.category {
+    list-style: none;
+    font-size: 0.9rem;
+    display: flex;
+    justify-content: space-around;
+    width: 30%;
+    padding-top: 0.2rem;
+}
+
+.category_li {
+    text-decoration: none;
+    color: grey;
+    letter-spacing: 0.1rem;
+    margin: auto;
+}
+
+.coffeeType {
+    background-color: #f1f1ef;
+    width: 90%;
+    margin: 1.5% auto;
+}
+
+.coffeeTypeSpan {
+    margin-left: 2%;
+    font-size: 1rem;
+}
+
+#coffee_icon {
+    font-size: 1.2rem;
+    margin-left: 0.5%;
+    margin-right: 0.2%;
+    padding: 1% 1% 1% 1%;
```

```suggestion
    padding: 1%;
```

이렇게 간단하게 나타내는 게 좋지 않을까요?